### ISSUE TITLE
Vue not working with reactivity

### ISSUE THREAD

**Issue by degustadev** [docsifyjs/docsify#790]
Hi!

I tried adding the following:

```HTML
<div id="example-1">
  <button v-on:click="counter += 1">Add 1</button>
  <p>The button above has been clicked {{ counter }} times.</p>
</div>
```
```JS
new Vue({
  el: '#example-1',
  data: {
    counter: 0
  }
})
```

I added `<script src="//unpkg.com/vue/dist/vue.min.js"></script>` to `index.html` and `executeScript: true`.

But it doesn't work, documentation says Docsify supports Vue tho.

**Comment by degustadev** [comment]
Related to: https://github.com/docsifyjs/docsify/issues/788

**Comment by BillyLuo** [comment]
same question

**Comment by jpkraemer** [comment]
does not work for me either

**Comment by jhildenbiddle** [comment]
@degustadev --

Fixed in #1271. The example code you provided has been tested and works as-is.
